lib/mgmt_msg_native.h

```diff
@@ -77,6 +77,11 @@ extern "C" {
  * mgmt_msg_native_get_msg_len() - Get the total length of the msg.
  * mgmt_msg_native_send_msg() - Send the message.
  *
+ * mgmt_msg_native_xpath_data_encode() - Encode xpath in xpath, data format message.
```

Comment:
There's no such function and it mentioned in several places in this header.

Reply:
Sigh.. yes, I did a search replace, odd that it missed some.